it/uk/gov/hmrc/emailverification/EmailVerificationISpec.scala

```diff
@@ -15,56 +14,64 @@ class EmailVerificationISpec extends IntegrationBaseSpec with GivenWhenThen {
+        await(ws.url(appClient("/verified-email-addresses")).post(Json.obj("token" -> token2))).status shouldBe 201
+      }
     }
 
     "second verification request should return successful 204 response" in new Setup {
       Given("The email service is running")
       stubSendEmailRequest(202)
 
       When("client submits a verification request")
-      val response1 = appClient("/verification-requests").post(verificationRequest(emailToVerify, templateId, continueUrl)).futureValue
-      response1.status shouldBe 201
-      val token = decryptedToken(lastVerificationEMail)._1.get
-
-      Then("the verification request with the token should be successful")
-      appClient("/verified-email-addresses").post(Json.obj("token" -> token)).futureValue.status shouldBe 201
-      Then("an additional verification requests with the token should be successful, but return with a 204 response")
-      appClient("/verified-email-addresses").post(Json.obj("token" -> token)).futureValue.status shouldBe 204
+      withClient { ws =>
+        val response1 = await(ws.url(appClient("/verification-requests")).post(verificationRequest(emailToVerify, templateId, continueUrl)))
+        response1.status shouldBe 201
+        val token = decryptedToken(lastVerificationEMail)._1.get
+
+        Then("the verification request with the token should be successful")
+        await(ws.url(appClient("/verified-email-addresses")).post(Json.obj("token" -> token))).status shouldBe 201
+        Then("an additional verification requests with the token should be successful, but return with a 204 response")
+          await(ws.url(appClient("/verified-email-addresses")).post(Json.obj("token" -> token))).status shouldBe 204
+      }
     }
 
     "email verification for two different emails should be successful" in new Setup {
       def submitVerificationRequest(emailToVerify: String, templateId: String, continueUrl: String) = {
-        val response = appClient("/verification-requests").post(verificationRequest(emailToVerify, templateId, continueUrl)).futureValue
-        response.status shouldBe 201
-        val token = decryptedToken(lastVerificationEMail)._1.get
-        And("the client verifies the token")
-        appClient("/verified-email-addresses").post(Json.obj("token" -> token)).futureValue.status shouldBe 201
-        Then("the email should be verified")
-        appClient(s"/verified-email-check").post(Json.obj("email" -> emailToVerify)).futureValue.status shouldBe 200
+        withClient { ws =>
+          val response = await(ws.url(appClient("/verification-requests")).post(verificationRequest(emailToVerify, templateId, continueUrl)))
+          response.status shouldBe 201
+          val token = decryptedToken(lastVerificationEMail)._1.get
+          And("the client verifies the token")
+          await(ws.url(appClient("/verified-email-addresses")).post(Json.obj("token" -> token))).status shouldBe 201
```

Comment:
Same goes for the the bit that starts with await and goes up to '.status'. Could reduce code usage here too.